tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/_util.tpl

```diff
@@ -27,16 +27,19 @@ Fields added to global context and returned:
 .Stress.Scenario - A `.` value passed down from a range loop over a scenarios list
             from values.yaml or a default value "stress".
 .Stress.ResourceGroupName - A pre-calculated resource group name value that can be passed down to various configurations that require it.
+.Stress.BaseName - A six character unique id derived from .Stress.ResourceGroupName that can be used in contexts with strict naming restrictions.
 
 See https://github.com/Masterminds/sprig/tree/master/docs for template function reference
 */}}
 {{- define "stress-test-addons.util.mergeStressContext" -}}
 {{- /* Copy scenario name into top level keys of global context */}}
 {{- $_global := index . 0 -}}
 {{- $_scenario := index . 1 -}}
-{{ $resourceGroupName := lower (print $_global.Release.Namespace "-" $_scenario "-" $_global.Release.Name "-" $_global.Release.Revision) }}
+{{- $uniqueTestName := lower (print $_global.Release.Namespace "-" $_scenario "-" $_global.Release.Name "-" $_global.Release.Revision) -}}
```

Comment:
Is there a particular reason why we are changing the name? Since the only place that uses it are the ResourceGroupName in the .Stress dictrionary

Reply:
I'll change it back. It made more sense to me to change the underlying variable when I was also consuming it to generate the `$uniqueTestId`.